src/design-system/README.md

```diff
@@ -295,6 +295,22 @@ export default () => (
 );
 ```
 
+#### DebugLayout
```

Comment:
Starting to feel like this readme is redundant now given we now have the web cheat sheet. Should we point the readme to it instead?

Reply:
Yeah, good point. Update it to only include instructions on running the docs locally?

Reply:
I've pushed an update, let me know what you think.